### ISSUE TITLE
Storybook 6.0

### ISSUE THREAD

**Issue by lifeiscontent** [pocka/storybook-addon-designs#67]
Hey @pocka does this addon work with 6.0 yet?

**Comment by schonarth** [comment]
I just waited to install the addon after upgrading Storybook, and now it won't work. I'm not sure if I'm doing something wrong or if it isn't compatible.

**Comment by pocka** [comment]
Yes, as described in README, this addon works with Storybook v5.0.0 and later.

If you encounter a bug, please file an issue with versions of Storybook and the addon you're using.
Since there are no issue templates yet, these would be helpful:

- Versions
  - `@storybook/<framework>`
  - `storybook-addon-designs`
  - Other addons if needed
- Reproduction repository (CodeSandbox is okay too) or Configuration
- Is there an error OR the addon does not appear without any errors?

**Comment by schonarth** [comment]
I got it. Had to change the `myStory.story = { parameters: { ...` for the `Basic.parameters = { ...` syntax. Then it worked.

Thanks!

**Comment by pocka** [comment]
Ah, I missed that! I'll update README and docs later.

**Comment by JoshRobertson** [comment]
@pocka can there be an update to the peer dependencies to  support` storybook ^5.0.0 || ^6.0.0` to remove the warnings when installing with Storybook 6?

```
warning " > storybook-addon-designs@5.4.1" has incorrect peer dependency "@storybook/addons@^5.0.0".
warning " > storybook-addon-designs@5.4.1" has incorrect peer dependency "@storybook/components@^5.0.0".
warning " > storybook-addon-designs@5.4.1" has incorrect peer dependency "@storybook/theming@^5.0.0".
```

**Comment by pocka** [comment]
@JoshRobertson Thank you for pointing out!

I've just published [v5.4.2](https://github.com/pocka/storybook-addon-designs/releases/tag/v5.4.2) that uses `^5.0.0 || ^6.0.0` query for the Storybook modules in `peerDependencies`.
